lib/clients/metrics.ts

```diff
@@ -15,28 +31,77 @@ export class Metrics {
       flushIntervalSeconds: 5,
     });
 
+    const port = config.PORT_METRICS ? parseInt(config.PORT_METRICS, 10) : PrometheusExporter.DEFAULT_OPTIONS.port;
+
+    this.exporter = new PrometheusExporter({ port }, () => {
+      log.info(`[metrics] exporter ready ${log.vars({ port, path: PrometheusExporter.DEFAULT_OPTIONS.endpoint })}`);
+    });
+
+    this.meter = new MeterProvider({ exporter: this.exporter, interval: 1000 }).getMeter('alexa-runtime');
+
+    this.counters = {
+      alexaError: this.meter.createCounter('alexa_request_error', { description: 'Alexa requests errors' }),
+      alexaInvocation: this.meter.createCounter('alexa_invocation', { description: 'Alexa invocations' }),
+      alexaRequest: this.meter.createCounter('alexa_request', { description: 'Alexa requests' }),
+      httpRequest: this.meter.createCounter('http_request', { description: 'HTTP requests' }),
+    };
+
+    this.recorders = {
+      httpRequestDuration: this.meter.createValueRecorder('http_request_duration', { description: 'Http requests duration' }),
+    };
+
     this.hashids = config.CONFIG_ID_HASH ? new Hashids(config.CONFIG_ID_HASH, 10) : null;
   }
 
   request() {
     this.client.increment('alexa.request');
+
+    this.counters.alexaRequest.bind(this.labels).add(1);
```

Comment:
the `.bind()` method has this comment:

> Returns a Instrument associated with specified Labels. It is recommended to keep a reference to the Instrument instead of always calling this method for every operations.

are we supposed to be calling `.bind()` each time one of these methods is run or is there a different way we should be doing it?